lib/logidze/model.rb

```diff
@@ -96,10 +96,10 @@ def diff_from(ts)
 
     # Restore record to the previous version.
     # Return false if no previous version found, otherwise return updated record.
-    def undo!
+    def undo!(as_new: nil)
```

Comment:
I didn't want to write `Logidze.preserve_future` twice (see `switch_to` below).  But maybe you're right.